src/tagging/tests/__snapshots__/tag.modifier.test.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Tagging modifier match snapshot 1`] = `
-<Fragment>
+<React.Fragment>
```

Comment:
looks like this change is superfluous (and causes the new failure)

Reply:
That's that the snapshot update command did. Shall I revert it?

Reply:
It's probably about a version of something :-(

Reply:
Anyway did that.